Remove more AppleWin code in WebCore

fujii opened the pull request:
#### 27a3cbeaf3eeb0543b10bb29641c33e6673d3369
<pre>
Remove more AppleWin code in WebCore
<a href="https://bugs.webkit.org/show_bug.cgi?id=253396">https://bugs.webkit.org/show_bug.cgi?id=253396</a>

Reviewed by Don Olmstead.

* Source/WebCore/PlatformWin.cmake:
* Source/WebCore/platform/graphics/BifurcatedGraphicsContext.cpp:
(WebCore::BifurcatedGraphicsContext::deprecatedPrivateContext const): Deleted.
* Source/WebCore/platform/graphics/BifurcatedGraphicsContext.h:
* Source/WebCore/platform/graphics/Font.h:
(WebCore::Font::scriptCache const):
* Source/WebCore/platform/graphics/GraphicsContext.h:
(WebCore::GraphicsContext::deprecatedPrivateContext const): Deleted.
* Source/WebCore/platform/graphics/NullGraphicsContext.h:
* Source/WebCore/platform/graphics/avfoundation/MediaPlayerPrivateAVFoundation.cpp:
(WebCore::MediaPlayerPrivateAVFoundation::scheduleMainThreadNotification):
* Source/WebCore/platform/graphics/ca/GraphicsLayerCA.cpp:
(WebCore::GraphicsLayerCA::getDebugBorderInfo const):
* Source/WebCore/platform/graphics/win/DIBPixelData.h:
* Source/WebCore/platform/graphics/win/FontCacheWin.cpp:
(WebCore::FontCache::platformInit):
(WebCore::FontCache::createFontPlatformData):
* Source/WebCore/platform/graphics/win/FullScreenWindow.cpp:
(WebCore::FullScreenWindow::createWindow):
(WebCore::FullScreenWindow::wndProc):
(WebCore::FullScreenWindow::setRootChildLayer): Deleted.
* Source/WebCore/platform/graphics/win/FullScreenWindow.h:
(WebCore::FullScreenWindow::rootChildLayer const): Deleted.
* Source/WebCore/platform/graphics/win/GraphicsContextWin.cpp:
(WebCore::GraphicsContext::getWindowsContext):
(WebCore::GraphicsContextPlatformPrivate::save): Deleted.
(WebCore::GraphicsContextPlatformPrivate::restore): Deleted.
(WebCore::GraphicsContextPlatformPrivate::clip): Deleted.
(WebCore::GraphicsContextPlatformPrivate::scale): Deleted.
(WebCore::GraphicsContextPlatformPrivate::rotate): Deleted.
(WebCore::GraphicsContextPlatformPrivate::translate): Deleted.
(WebCore::GraphicsContextPlatformPrivate::concatCTM): Deleted.
(WebCore::GraphicsContextPlatformPrivate::setCTM): Deleted.
* Source/WebCore/platform/graphics/win/GraphicsContextWin.h: Removed.
* Source/WebCore/platform/graphics/win/MediaPlayerPrivateMediaFoundation.cpp:
(WebCore::MediaPlayerPrivateMediaFoundation::Direct3DPresenter::paintCurrentFrame):
* Source/WebCore/platform/graphics/win/SimpleFontDataCairoWin.cpp:
(WebCore::Font::platformInit):
* Source/WebCore/platform/graphics/win/SimpleFontDataWin.cpp:
(WebCore::Font::platformDestroy):
(WebCore::Font::setShouldApplyMacAscentHack): Deleted.
(WebCore::Font::shouldApplyMacAscentHack): Deleted.
(WebCore::Font::ascentConsideringMacAscentHack): Deleted.
(WebCore::Font::initGDIFont): Deleted.
(WebCore::Font::boundsForGDIGlyph const): Deleted.
(WebCore::Font::widthForGDIGlyph const): Deleted.
(WebCore::Font::scriptFontProperties const): Deleted.
* Source/WebCore/platform/win/BString.cpp:
* Source/WebCore/platform/win/BString.h:
* Source/WebCore/platform/win/DragImageWin.cpp:
(WebCore::createDragImageForLink):
* Source/WebCore/platform/win/PlatformScreenWin.cpp:
* Source/WebCore/platform/win/PopupMenuWin.cpp:
(WebCore::PopupMenuWin::paint):
* Source/WebCore/platform/win/WebCoreBundleWin.cpp:
(WebCore::createWebKitBundle): Deleted.
(WebCore::webKitBundle): Deleted.
* Source/WebCore/platform/win/WebCoreBundleWin.h:
* Source/WebKit/UIProcess/win/WebPopupMenuProxyWin.cpp:
(WebKit::WebPopupMenuProxyWin::paint):
* Source/cmake/WinTools.make: Removed.

Canonical link: <a href="https://commits.webkit.org/261288@main">https://commits.webkit.org/261288@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/3127ebb172fece2771ac4b3e494b60ffb04ee165

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/6/builds/110969 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/77/builds/20109 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/43/builds/43536 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/87/builds/2397 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/12/builds/119815 "Built successfully") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/11/builds/114928 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/76/builds/21484 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/85/builds/11210 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/86/builds/2060 "Passed tests") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/19/builds/116722 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/78/builds/15957 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/3/builds/99140 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/36/builds/103450 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/9/builds/97918 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/73/builds/30804 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/35/builds/44388 "Passed tests") | 
| [✅ 🛠 🧪 jsc](https://ews-build.webkit.org/#/builders/1/builds/99575 "Built successfully and passed tests") | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/81/builds/12628 "Built successfully") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/70/builds/32140 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/34/builds/86306 "Passed tests") | 
| [✅ 🛠 🧪 jsc-arm64](https://ews-build.webkit.org/#/builders/83/builds/10744 "Built successfully and passed tests") | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/82/builds/13175 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/84/builds/9121 "Passed tests") | [✅ 🛠 jsc-armv7](https://ews-build.webkit.org/#/builders/38/builds/100695 "Built successfully") | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/80/builds/18586 "Built successfully") | [  ~~🧪 mac-wk2-stress~~](https://ews-build.webkit.org/#/builders/62/builds/51760 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🧪 jsc-armv7-tests](https://ews-build.webkit.org/#/builders/46/builds/31392 "Passed tests") | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/74/builds/7824 "Built successfully and passed tests") | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/79/builds/15122 "Built successfully") | | [✅ 🛠 jsc-mips](https://ews-build.webkit.org/#/builders/37/builds/108732 "Built successfully") | 
| | | | [✅ 🧪 jsc-mips-tests](https://ews-build.webkit.org/#/builders/45/builds/26794 "Passed tests") | 
<!--EWS-Status-Bubble-End-->

donny-dont commented on Source/WebCore/platform/graphics/win/DIBPixelData.h:
```diff
@@ -29,11 +29,7 @@
-#if USE(CG)
-#include <CoreFoundation/CFBase.h>
-#else
 typedef unsigned char UInt8;
```
Do we need to keep this `typedef`?

fujii replied:
Good catch. I'd like to split a patch. Will do it as a follow-up.

donny-dont reviewed (approved):
LGTM minus the one question.

brentfulgham reviewed (approved):
What a lot of great cleanup! Looks excellent.

fujii committed:
Remove more AppleWin code in WebCore
https://bugs.webkit.org/show_bug.cgi?id=253396

Reviewed by Don Olmstead.

* Source/WebCore/PlatformWin.cmake:
* Source/WebCore/platform/graphics/BifurcatedGraphicsContext.cpp:
(WebCore::BifurcatedGraphicsContext::deprecatedPrivateContext const): Deleted.
* Source/WebCore/platform/graphics/BifurcatedGraphicsContext.h:
* Source/WebCore/platform/graphics/Font.h:
(WebCore::Font::scriptCache const):
* Source/WebCore/platform/graphics/GraphicsContext.h:
(WebCore::GraphicsContext::deprecatedPrivateContext const): Deleted.
* Source/WebCore/platform/graphics/NullGraphicsContext.h:
* Source/WebCore/platform/graphics/avfoundation/MediaPlayerPrivateAVFoundation.cpp:
(WebCore::MediaPlayerPrivateAVFoundation::scheduleMainThreadNotification):
* Source/WebCore/platform/graphics/ca/GraphicsLayerCA.cpp:
(WebCore::GraphicsLayerCA::getDebugBorderInfo const):
* Source/WebCore/platform/graphics/win/DIBPixelData.h:
* Source/WebCore/platform/graphics/win/FontCacheWin.cpp:
(WebCore::FontCache::platformInit):
(WebCore::FontCache::createFontPlatformData):
* Source/WebCore/platform/graphics/win/FullScreenWindow.cpp:
(WebCore::FullScreenWindow::createWindow):
(WebCore::FullScreenWindow::wndProc):
(WebCore::FullScreenWindow::setRootChildLayer): Deleted.
* Source/WebCore/platform/graphics/win/FullScreenWindow.h:
(WebCore::FullScreenWindow::rootChildLayer const): Deleted.
* Source/WebCore/platform/graphics/win/GraphicsContextWin.cpp:
(WebCore::GraphicsContext::getWindowsContext):
(WebCore::GraphicsContextPlatformPrivate::save): Deleted.
(WebCore::GraphicsContextPlatformPrivate::restore): Deleted.
(WebCore::GraphicsContextPlatformPrivate::clip): Deleted.
(WebCore::GraphicsContextPlatformPrivate::scale): Deleted.
(WebCore::GraphicsContextPlatformPrivate::rotate): Deleted.
(WebCore::GraphicsContextPlatformPrivate::translate): Deleted.
(WebCore::GraphicsContextPlatformPrivate::concatCTM): Deleted.
(WebCore::GraphicsContextPlatformPrivate::setCTM): Deleted.
* Source/WebCore/platform/graphics/win/GraphicsContextWin.h: Removed.
* Source/WebCore/platform/graphics/win/MediaPlayerPrivateMediaFoundation.cpp:
(WebCore::MediaPlayerPrivateMediaFoundation::Direct3DPresenter::paintCurrentFrame):
* Source/WebCore/platform/graphics/win/SimpleFontDataCairoWin.cpp:
(WebCore::Font::platformInit):
* Source/WebCore/platform/graphics/win/SimpleFontDataWin.cpp:
(WebCore::Font::platformDestroy):
(WebCore::Font::setShouldApplyMacAscentHack): Deleted.
(WebCore::Font::shouldApplyMacAscentHack): Deleted.
(WebCore::Font::ascentConsideringMacAscentHack): Deleted.
(WebCore::Font::initGDIFont): Deleted.
(WebCore::Font::boundsForGDIGlyph const): Deleted.
(WebCore::Font::widthForGDIGlyph const): Deleted.
(WebCore::Font::scriptFontProperties const): Deleted.
* Source/WebCore/platform/win/BString.cpp:
* Source/WebCore/platform/win/BString.h:
* Source/WebCore/platform/win/DragImageWin.cpp:
(WebCore::createDragImageForLink):
* Source/WebCore/platform/win/PlatformScreenWin.cpp:
* Source/WebCore/platform/win/PopupMenuWin.cpp:
(WebCore::PopupMenuWin::paint):
* Source/WebCore/platform/win/WebCoreBundleWin.cpp:
(WebCore::createWebKitBundle): Deleted.
(WebCore::webKitBundle): Deleted.
* Source/WebCore/platform/win/WebCoreBundleWin.h:
* Source/WebKit/UIProcess/win/WebPopupMenuProxyWin.cpp:
(WebKit::WebPopupMenuProxyWin::paint):
* Source/cmake/WinTools.make: Removed.

Canonical link: https://commits.webkit.org/261288@main
